SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -235,6 +235,7 @@
 
     <WriteProjectInfoFile ProjectName="$(MSBuildProjectName)"
        FullProjectPath="$(MSBuildProjectFullPath)"
+       ProjectLanguage="$(ProjectLanguage)"
```

Comment:
FYI I later found out that `$(Language)` is available out of the box, and is set to either `C#` or `VB` - I don't think it's worth the change though